src/main/java/com/ucm/shiro/facturae/v3_1/transformer/FacturaeTransformerToV3_2.java

```diff
@@ -0,0 +1,13 @@
+package com.ucm.shiro.facturae.v3_1.transformer;
+
+import com.ucm.shiro.facturae.agnostic.transformer.AbstractFacturaeTransformer;
+import com.ucm.shiro.facturae.exception.FacturaeTransformException;
+
+public class FacturaeTransformerToV3_2 extends AbstractFacturaeTransformer {
+
+    private static final String FACTURAE_TO_3_2_XSLT_PATH = "xml/schema/facturae/v3_1/Transformer3.1To3.2.xslt";
```

Comment:
Now the transformer transfrom from 3.0 to 3.2.1.

Reply:
With the current approach, the transform is ok